frontend/packages/knative-plugin/src/components/overview/KnativeResourceOverviewPage.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import { GraphElement } from '@patternfly/react-topology';
```

Comment:
It's possible that this import is fetching the react-topology in main vendor build and that's why it's exceeding 3.50mb limit